HomeWork3.1/HomeWork3.1/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using HomeWork3._1;
+using HomeWork3._1.Collection.Abstractions;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using HomeWork3._1.Collection;
+
+void ConfigureService(IServiceCollection serviceCollection)
+{
+    serviceCollection
+        .AddScoped<IPersonalisedCollection<string>, PersonalisedCollection<string>>()
```

Comment:
.AddScoped<IPersonalisedCollection<T>, PersonalisedCollection<T>>() is more preferable